JULABO/iocBoot/iocJULABO-IOC-01/config.xml

```diff
@@ -8,6 +8,7 @@
 <macro name="READ_EXT_TEMP_CMD_NUMBER" pattern="^[0-9]+$" description="Command number for reading external temperature. The prefix 'IN_PV_' is added automatically. Defaults to 02" />
 <macro name="READ_HIGH_LIM_CMD_NUMBER" pattern="^[0-9]+$" description="Command number for reading the high limit. The prefix 'IN_SP_' is added automatically. Defaults to 01" />
 <macro name="READ_LOW_LIM_CMD_NUMBER" pattern="^[0-9]+$" description="Command number for reading the low limit. The prefix 'IN_SP_' is added automatically. Defaults to 02" />
+<macro name="FLOWCONTROL" pattern="^Y|N$" description="Sets Julabo to use software flow control. Defaults to N." />
```

Comment:
As both software and hardware flow control are possible, I think it would be better to use:

XONXOFF   - enable or disable software flow control
CRTSCTS - enable or disable hardware flow control

We also need to check the julabo settings - I though the one on ZOOM was on software flow control,I though they had either software or hardware on the menu (i.e. no flow control was not an option)

Reply:
The Keithley 2400 and 2700 has used the name SOFTFLOWCNTL - i guess we should be consistent ans use this and HWFLOWCNTR as the macro names